lib/integer.gi

```diff
@@ -1195,7 +1195,11 @@ InstallGlobalFunction(PowerModInt,function ( r, e, m )
     # return the power
     return pow;
 end);
-
+if IsBound(POWERMODINT) then
```

Comment:
Similar for other functions that follow a similar pattern. If they are only used for testing (and I do like testing by comparing two implementations), perhaps they could be moved to a file in tst?